build.sh

```diff
@@ -364,7 +364,7 @@ fi
 if [ $__CrossBuild == 1 ]; then
     export CROSSCOMPILE=1
     if ! [[ -n "$ROOTFS_DIR" ]]; then
-        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs/$__BuildArch"
+        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs"
     fi
```

Comment:
export in a script shouldn't affect the global environment. I haven't tried here.